source/process/security.md

```diff
@@ -75,6 +75,8 @@ The following resources are monitoring for information about new security threat
 - http://www.cert.org/
 - https://www.reddit.com/r/netsec/
 
+Moreover, all third-party dependancies are updated for potential security issues. These reviews include, but are not limited to, the following repositories: [/mattermost-server](https://github.com/mattermost/mattermost-server), [/mattermost-webapp](https://github.com/mattermost/mattermost-webapp), [/desktop](https://github.com/mattermost/desktop), [/mattermost-mobile](https://github.com/mattermost/mattermost-mobile) and [/mattermost-redux](https://github.com/mattermost/mattermost-redux).
```

Comment:
How about "All dependencies are updated on a regular basis to make sure we have the latest security updates." Not sure why we then need to enumerate some repositories.